test/benchmark/test/__snapshots__/reporter.test.ts.snap

```diff
@@ -0,0 +1,14 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
+
+exports[`summary 1`] = `
+"
+
+  good - summary.bench.ts > suite-a
+    ?.??x faster than bad
```

Comment:
It looks like this test might be flaky since "good" one becomes really slow depending on CI workload
- https://github.com/vitest-dev/vitest/actions/runs/8568796944/job/23483403713#step:9:320